t/shared.t

```diff
@@ -33,5 +32,23 @@ is( $hash{multi}->{key1},      "foo", "multidimension, key1" );
 is( $hash{multi}->{arr1}->[0], "bar", "multidimension, arr1 - key0" );
 is( $hash{multi}->{arr1}->[1], "baz", "multidimension, arr1 - key1" );
 
-unlink("vars.db");
-unlink("vars.db.lock");
+{
+    @array = (0);
+    my @pids;
+    for my $i (0..5) {
+        my $pid = fork();
+        if ($pid == 0) {
+            # child
+            sleep 1;
```

Comment:
Thats an awesome idea.  I did not know about `pause()`.  However your example code has a bug.  Should be
```
$SIG{CONT} = \&push_and_exit;
```
instead of
```
$SIG{CONT = &push_and_exit;
```

The problem is when I run this code it hangs.  Can't figure out why.  :rage1: